src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -40,6 +40,8 @@ public class SqlManagementTestUtilities
 
         public const string DefaultEuapPrimaryLocation = "East US 2 EUAP";
 
+        public const string DefaultEuapPrimaryLocationId = "eastus2euap";
+
         public const string DefaultLogin = "dummylogin";
```

Comment:
Let's make this a separate issue because it's orthogonal to the vnet test that @dnayantara is fixing